tests/Feature/GraphQL/Queries/GetWalletTest.php

```diff
@@ -405,6 +406,17 @@ public function test_it_can_get_a_wallet_and_filter_token_accounts(): void
             'reservedBalance' => $this->tokenAccount->reserved_balance,
             'isFrozen' => $this->tokenAccount->is_frozen,
         ], $response['tokenAccounts']['edges'][0]['node']);
+
+        $response = $this->graphql($this->method, [
+            'id' => $id = $this->wallet->id,
+            'bulkFilter' => [
+                [
+                    'collectionId' => $this->collection->collection_chain_id,
+                    'tokenIds' => [$this->token->token_chain_id],
```

Comment:
Should we also add a test using a integer ranges as well?

Reply:
Yes can be done